pydm/display.py

```diff
@@ -188,13 +185,8 @@ def load_py_file(pyfile, args=None, macros=None):
     # submodules can be loaded.
     # Eventually, this should go away, and intelligence modules should behave
     # as real python modules.
-    module_dir = os.path.dirname(os.path.abspath(pyfile))
-    if module_dir not in sys.path:
-        sys.path.append(module_dir)
-    temp_name = str(uuid.uuid4())
+    module = import_module_by_filename(os.path.abspath(pyfile))
 
-    # Now load the intelligence module.
-    module = imp.load_source(temp_name, pyfile)
```

Comment:
:+1: good to do these sorts of cleanup

Reply:
This is also the subject of the issue #846 - a noisy deprecation.